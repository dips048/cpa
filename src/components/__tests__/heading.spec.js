import React from "react"
import renderer from "react-test-renderer"
import Heading from "../heading"
import { render } from "@testing-library/react"

describe("Heading", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Heading title={"About"} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  test("should render given `title`" , () => {
    const { getByText } = render(<Heading title={"About"} />)
    expect(getByText("About")).toBeInTheDocument()
  })
})
