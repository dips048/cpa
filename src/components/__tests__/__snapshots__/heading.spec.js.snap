// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders correctly 1`] = `
<div
  className="heading"
>
  <div
    className="container"
  >
    <div
      className="row py-5"
    >
      <h1>
        About
      </h1>
    </div>
  </div>
</div>
`;
